src/AgentContainer.cpp

```diff
@@ -656,15 +656,15 @@ void AgentContainer::initAgentsCensus (iMultiFab& num_residents,    /*!< Number
                 work_i_ptr[ip] = i;
                 work_j_ptr[ip] = j;
                 nborhood_ptr[ip] = nborhood;
-                work_nborhood_ptr[ip] = 5*nborhood;
```

Comment:
What was `work_nborhood_ptr` being set to `5*nborhood` originally?

Reply:
Yes, but wasn't used anywhere after this initialization. I didn't understand the `5*nborhood`

Reply:
@atmyers Do you know why?